docs/source/elastic/run.rst

```diff
@@ -1,4 +1,4 @@
-.. _launcher-api:
+.. _launcher-ddp-api:
 
 torchrun (Elastic Launch)
```

Comment:
this needs to change to `torch.distributed.run (Elastic Launch)` since we are making torchrun point to `torch.utils.run` now. You'll also have to edit:

1. https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py#L19
2.  https://github.com/pytorch/pytorch/blob/master/torch/distributed/launch.py#L7